session/src/main/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandler.java

```diff
@@ -30,9 +33,7 @@
 import uk.gov.di.ipv.cri.common.library.util.EventProbe;
 
 import java.time.Clock;
-import java.util.Map;
-import java.util.TreeMap;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Please could you adjust see:  https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/java.html#imports

Reply:
Removed wildcard